ui/src/app/auth/Login.tsx

```diff
@@ -1,183 +1,12 @@
-              </h2>
-            </div>
-            <div className="mt-8 max-w-sm sm:mx-auto sm:w-full md:max-w-lg">
-              <div className="px-4 py-8 sm:px-10">
-                {providers && providers.length > 0 && (
-                  <div className="mt-6 flex flex-col space-y-5">
-                    {providers.map((provider) => (
-                      <div key={provider.name}>
-                        <a
-                          href="#"
-                          className="bg-white text-gray-500 border-gray-300 inline-flex w-full justify-center rounded-md border px-4 py-2 text-sm font-medium shadow-sm hover:text-violet-500 hover:shadow-violet-300"
-                          onClick={(e) => {
-                            e.preventDefault();
-                            authorize(provider.authorize_url);
-                          }}
-                        >
-                          <span className="sr-only">
-                            Sign in with {provider.name}
-                          </span>
-                          <FontAwesomeIcon
-                            icon={provider.icon}
-                            className="text-gray h-5 w-5"
-                            aria-hidden={true}
-                          />
-                          <span className="ml-2">With {provider.name}</span>
-                        </a>
-                      </div>
-                    ))}
-                  </div>
-                )}
-                {(!providers || providers.length === 0) && (
-                  <div className="bg-white shadow sm:rounded-lg">
-                    <div className="px-4 py-5 sm:p-6">
-                      <h3 className="text-gray-900 text-base font-semibold leading-6">
-                        No Providers
-                      </h3>
-                      <div className="text-gray-500 mt-2 max-w-xl text-sm">
-                        <p>
-                          Authentication is set to{' '}
-                          <span className="font-medium">required</span>,
-                          however, there are no login providers configured.
-                          Please see the documentation for more information.
-                        </p>
-                      </div>
-                      <div className="mt-3 text-sm leading-6">
-                        <a
-                          href="https://www.flipt.io/docs/configuration/authentication#method-oidc"
-                          className="text-violet-600 font-semibold hover:text-violet-500"
-                        >
-                          Configuring Authentication
-                          <span aria-hidden="true"> &rarr;</span>
-                        </a>
-                      </div>
-                    </div>
-                  </div>
-                )}
-              </div>
-            </div>
-          </div>
-        </main>
-      </div>
-    </>
+    <NotificationProvider>
+      <LoginWrapper />
```

Comment:
I would just do all this in a single file IMO no need for `LoginWrapper`, could just do like we do in `Layout.tsx`:

```jsx
function InnerLogin() {
 // the contents of LoginWrapper here
}

export default function Login() {
return (
  <NotificationProvider>
    <InnerLogin />
    <ErrorNotification />
  </NotificationProvider>
)
}
```